lightning/src/util/events.rs

```diff
@@ -246,6 +260,13 @@ impl Writeable for Event {
 					(2, claim_from_onchain_tx, required),
 				});
 			},
+			&Event::MPPFragmentFailed { ref payment_hash, ref payment_path } => {
+				8u8.write(writer)?;
```

Comment:
I feel like this should be odd. Old clients should be able to ignore `MPPFragmentFailed` events - they won't retry anyway so its not like they aren't broken in *new* ways, they're just broken, and shouldn't *also* fail to even run.